OuterWildsRandomSpeedrun/Marshmallow.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using UnityEngine;
+
+namespace OuterWildsRandomSpeedrun
+{
+    public class Marshmallow : MonoBehaviour
+    {
+        float z;
+
+        public void OnTriggerEnter(Collider collider)
+        {
+            if (collider == Locator.GetPlayerCollider())
+                Collect();
+        }
+
+        public void Collect()
+        {
+            OnCollected?.Invoke();
+        }
+
+        public void Update()
+        {
+            z += Time.deltaTime * 48f;
```

Comment:
Why are we constantly increasing the z value?

Reply:
it's sort of a compromise between readability and performance, since we're setting the rotation with a vector instead of directly. the math to calculate the rotation value is less concise, and setting the rotation directly is not really the way you usually do it in Unity, in part because it can't be delegated to the gpu as easily.